src/test/java/io/xstefank/wildfly/bot/utils/GitHubJson.java

```diff
@@ -0,0 +1,82 @@
+package io.xstefank.wildfly.bot.utils;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Class responsible for handling and changing properties of the GitHub JSON template file.
+ */
+public class GitHubJson {
+
+    private static String PULL_REQUEST = "pull_request";
+    private static String COMMITS = "commits";
+    private static String BODY = "body";
+    private static String TITLE = "title";
+    private static String SHA = "sha";
+    private static String HEAD = "head";
+    private static String ACTION = "action";
+
+    private static JsonNode file;
+    private static Builder builder;
+
+    private GitHubJson(Builder jsonHandlerBuilder) {
+        builder = jsonHandlerBuilder;
+        file = jsonHandlerBuilder.jsonFile;
+    }
+
+    public static Builder builder(String fileName) throws IOException {
```

Comment:
Here the javadoc would make sense because it's a public method and the fileName name isn't telling you where you will look for that file.